translations/fr/pack/rogue.json

```diff
@@ -0,0 +1,116 @@
+[
+   {
+      "code": "38001a",
+      "name": "Malicia",
+      "traits": "X-Men.",
+      "text": "<i>Contact Cutané</i> — <b>Action</b> : attachez Touché à un autre personnage. Vous gagnez tous les [[TRAITS]] du personnage attaché jusqu'à la fin du round. (Limite d'une fois par round.)\n<b>Réponse forcée</b> : après le début de la phase des Joueurs, trouvez et mettez de côté Touché."
+   },
+   {
+      "code": "38001b",
+      "name": "Anna Marie",
+      "flavor": "\"Vous ne pourriez pas vivre la vie qui est la mienne.\"",
+      "traits": "Mutant.",
+      "text": "<b>Mise en place</b> : mettez de côté votre amélioration Touché.\n<i>Retrait</i> — <b>Réponse forcée</b> : après être passé sous cette forme, mettez de côté Touché."
+   },
+   {
+      "code": "38002",
+      "name": "Touché",
+      "traits": "Condition.",
+      "text": "Si Touché est attachée à un :\nSbire — Les attaques de Malicia gagnent Déferlement.\nMéchant — Malicia gagne Riposte 1.\nAllié — Malicia gagne le trait [[AÉRIEN]].\nHéros — Malicia gagne Robuste."
+   },
+   {
+      "code": "38003",
+      "name": "Gambit",
+      "subname": "Remy Lebeau",
```

Comment:
Should this be `Remy LeBeau`? I found that value in the Gambit FR pack on a different PR.

Reply:
I'm sorry, but I don't understand your question. I checked and I see no mistakes (but maybe I'm wong) :
- This PR is for Rogue hero file and content seems ok
- Rogue's ally is Gambit, and it's card #3
- For this ally, I checked name (Gambit) and subname (Remy LeBeau) and they are ok in this file

So, for you, where is the problem ?

Reply:
Yes ! You're right (I missed it in all my controls ... ;).
=> Remy's lastname fixed